sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2DataFrameSuite.scala

```diff
@@ -141,4 +141,66 @@ class DataSourceV2DataFrameSuite extends QueryTest with SharedSQLContext with Be
       }
     }
   }
+
+  testQuietly("saveAsTable: table doesn't exist => create table") {
```

Comment:
just for curiosity, why do we need `testQuietly` here?

Reply:
It was before merging Gengliang's "Catalogs.load should work for inbuilt catalogs" PR. It may not be needed now, but I don't think it's a big deal